testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/management/cli/DeployAllDomainTestCase.java

```diff
@@ -178,20 +181,10 @@ public void testDeploymentLiveCycleWithServerGroups() throws Exception {
         ctx.handle("deployment disable --server-groups=" + sgOne + ' ' + cliTestApp1War.getName());
         ctx.handle("deployment disable --server-groups=" + sgTwo + ' ' + cliTestApp2War.getName());
 
-        // Step 3b) Try disabling application deployment in wrong server group space. expect command execution fail
```

Comment:
Does this change backward compatibility for previous server version? I'm not sure because these tests doesn't exist earlier.